build.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
 repositories {
     gradlePluginPortal()
     mavenCentral()
+    google()
```

Comment:
`l`: Just curios. Why do we need the Google repo?

Reply:
cause `sentry-android` pulls in some androidx/android deps, and if we don't add it, gradle doesn't know where to pull them from